### DISCUSSION TITLE
Local Multiplayer?

### DISCUSSION THREAD

**Discussion by ogami1972** [Drewol/unnamed-sdvx-clone#580] [Ideas]
Hello!

My family and I are in love with this game! Is there a way to have individual players have hi-scores for charts, like entering the players initials when they get the hi-score?

Thank you!

**Comment by ogami1972** [comment]
For the record, I'm willing to put a bounty on this feature ;)

**Reply by Drewol** [reply]
That's not necessary, a lot of the pieces for this are already in place and the work would mostly consist of tying those pieces together. I'll try to get to this feature after going through some of the already open Pull Requests.

**Reply by ogami1972** [reply]
Hello!

That's very kind of you :)

Now that I have had more time to get used to the nature of the game, I would like to "edit" the feature request:

I would like to see chart scores and skill analyzer results tied to profiles. I am not sure if it is possible, but actually tying "dan level" to profiles would be amazing. It may actually be that all this is already happening and I just haven't seen it yet (forest through the trees sort of thing, maybe). If so, please ignore, lol.

I think you understand what type of experience I am working towards. It really is an amazing game. Thanks for all you do and take care!

-ogami1972

**Reply by Drewol** [reply]
Opened an issue for tracking purposes #594

**Comment by ogami1972** [comment]
Hello!

Squee! I'm super excited!

Here's something interesting about the way I am currently handling multiplayer:

I have noticed, and previously commented, that memory usage seems to compound with play. In other words, you start off using 1.8gb of RAM, play a handful of charts, and find you are now using close to 5gb. In the past, this would just continue until all RAM was used and the game froze. This was eventually found to be a skin issue. In my experience, ALL 3rd party skins do this.

Tying the skin to the profile, and changing profiles from the settings menu, causes the game to itself to "cycle" ( I thought it was just reloading the skin until I realized it was checking for updates each time). This may be exclusive to the "EG" skin I am using. Regardless, it is a happy occurrence. 

TLDR: I played a few charts, and my RAM usage was around 5gb. Patty loaded her profile which caused the game to cycle and RAM usage dropped back to 1.8gb.

It should be noted that this DOES NOT occur if you change profile from the in-game menu (both FX buttons). You have to change profile in the Settings screen to trigger this cycle.

Interesting?

**Comment by ogami1972** [comment]
Hello!
 Related?:
Last night, I found that the game's Speed Mod could be linked to profiles, allowing Patty to use MMod while I use XMod. I did this by adding a line to the profile .cfg file. I wonder what other parameters can be specified there.

**Comment by Jaybberwocky** [comment]
Apologies (new here) - was this created/resolved? I'm looking to have separate users to create a local leaderboard. I saw a video poste by ogami1972 that showed it was possible, so I wanted to see whether or not it had been implemented and I just found it yet?
